app/src/sharedTest/java/org/oppia/app/walkthrough/WalkthroughActivityTest.kt

```diff
@@ -28,18 +36,28 @@ class OnboardingFragmentTest {
   @Test
   fun testWalkthroughFragment_increaseProgress_worksCorrectly() {
     ActivityScenario.launch(WalkthroughActivity::class.java).use {
-      onView(withId(R.id.next_button)).perform(click())
+      onView(withId(R.id.walkthrough_welcome_next_btn)).perform(click())
+      onView(withId(R.id.walkthrough_progress_bar)).check(matches(withProgress(2)))
+    }
+  }
+
```

Comment:
Please add test-cases for indicator position in page change.

Reply:
Can you explain a bit about what exactly do i need to do here